.github/workflows/helm-workflow.yaml

```diff
@@ -7,7 +7,9 @@ on:
     branches:
       - main
 env:
-  HELM_VERSION: v3.12.1
+  HELM_VERSION: v3.15.4
+  HELMFILE_VERSION: v0.167.1 
+  HELM_PLUGINS: >
```

Comment:
should here stand something?

Reply:
I've forgotten to remove it, now it should be gone